src/Tests/dotnet-watch.Tests/Watch/Utilities/WatchableApp.cs

```diff
@@ -95,12 +106,18 @@ public void Start(string projectRootPath, IEnumerable<string> arguments, string
 
             var commandSpec = new DotnetCommand(Logger, args.ToArray())
             {
-                WorkingDirectory = workingDirectory ?? projectRootPath,
+                WorkingDirectory = workingDirectory ?? projectDirectory,
             };
 
             commandSpec.WithEnvironmentVariable("DOTNET_USE_POLLING_FILE_WATCHER", "true");
             commandSpec.WithEnvironmentVariable("__DOTNET_WATCH_TEST_FLAGS", testFlags.ToString());
 
+            var encLogPath = Environment.GetEnvironmentVariable("HELIX_WORKITEM_UPLOAD_ROOT") is { } ciOutputRoot
+                ? Path.Combine(".hotreload", ciOutputRoot, asset.Name)
```

Comment:
This seems weird, are you prepending ".hotreload" to the front of `ciOutputRoot?  I think if `ciOutputRoot` is a rooted path then the first argument will just be ignored.

```suggestion
                ? Path.Combine(ciOutputRoot, asset.Name)
```

Reply:
Swapped `".hotreload"` and `ciOutputRoot`.